### ISSUE TITLE
View All Tab of different windows in the Open Tabs column

### ISSUE THREAD

**Issue by imdankwok** [one-tab-group/onetab.group#82]
It would be nice to view the Open Tabs of all windows instead of just the current one with the OneTabGroup pin tab opened.

**Comment by mmcintyre123** [comment]
I completely agree. This extension has so much going forward especially compared to similar extensions. But this is one major thing it is missing that every other inferior extension I have tested has.
